### PR TITLE
Adding missing RCC_CRRCR register

### PR DESCRIPTION
**Pull request by mathk**

### CODE REVIEW

**Comment by mathk** [comment]
Could most probably apply to other l4 family. Let me know the name of the peripheral/rcc file name if refactoring is needed.

**Comment by mathk** [comment]
CRRCR is only available for stm32l496  and stm32l4A6.

**Comment by mathk** [comment]
CRRCR is missing on stm32l4x5 and stm32l4x6

File: peripherals/rcc/rcc_crrcr.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+RCC:
```
[the remark this answers is not in the document]
Comment (mathk) [reply peripherals/rcc/rcc_crrcr.yaml]: File moved.

File: peripherals/rcc/rcc_l4.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+RCC:
+  CR:
+    MSIRANGE:
+      Range0: [0b0000, "range 0 around 100 kHz"]
```
[the remark this answers is not in the document]
Comment (mathk) [reply peripherals/rcc/rcc_l4.yaml]: yeap

**Comment by mathk** [comment]
@adamgreig I think this is now ready. Let me know if you need more change.

**Comment by twilco** [comment]
Thanks for this fix!  Has it been released yet?  I'm blocked by the same error as https://github.com/stm32-rs/stm32l4xx-hal/issues/38, which can't be fixed until a new version of this crate is cut with this fix.

**Comment by adamgreig** [comment]
@twilco this hasn't been released yet but will be in the next release 0.7.0, which I hope to cut once I finish the other pending reviews sometime in the next week.

**Comment by twilco** [comment]
Great, thanks @adamgreig!

**Comment by twilco** [comment]
Hey @adamgreig - any updates on when `0.7.0` is going to land? 🙂

**Pull request merged**